Makefile

```diff
@@ -9,14 +9,29 @@ endif
 tests/%.gfa:
 	curl -Lo ./$@ $(GFA_URL)/$*.gfa
 
+tests/%.og: tests/%.gfa
+	odgi build -g $< -o $@
+
 .PHONY: fetch
 fetch: $(TEST_FILES:%=tests/%.gfa)
 
+fetch-og: $(TEST_FILES:%=tests/%.og)
+
 .PHONY: test-slow-odgi
 test-slow-odgi: fetch
 	make -C slow_odgi test
 
 .PHONY: test-flatgfa
-test-flatgfa:
+test-flatgfa: fetch-og
```

Comment:
Might I suggest that we don't actually need to do this `.og` conversion? Seems like odgi works just fine reading directly from a GFA file, and eliminating this step would reduce the surface area of things that can break by one.

Reply:
Aah, makes sense! I left the `fetch-og` target because it is helpful when benchmarking